### PR TITLE
chore(ts): Convert SidebarPanel

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Review by billyvg — APPROVED** [review]
Hmm do we have any snapshots of this in getsentry too?

**Review by priscilawebdev — APPROVED** [review]
Looks Great 🎉

File: src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  right: -3px;
+  color: ${p => p.theme.gray3};
+
+  &:hover {
+    color: ${p => p.theme.gray5};
+  }
+`;
+
+const Title = styled('div')`
+  font-size: 18px;
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx]:
```suggestion
  font-size: ${p => p.theme.fontSizeExtraLarge};
```

File: src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: ${p => p.theme.sidebar.panel.headerHeight};
+  overflow: auto;
+`;
+
+const PanelClose = styled('a')`
+  float: right;
+  font-size: 22px;
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx]:
```suggestion
  font-size: ${p => p.theme.headerFontSize};
```

File: src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  box-shadow: 1px 0 2px rgba(0, 0, 0, 0.06);
+  text-align: left;
+  line-height: 24px;
+
+  ${getPositionForOrientation};
+`;
+
+const SidebarPanelHeader = styled('div')`
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+  padding: 20px;
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx]:
```suggestion
  padding: ${space(3)};
```

File: src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+          ? p.theme.sidebar.collapsedWidth
+          : p.theme.sidebar.expandedWidth};
+      `;
+
+const StyledSidebarPanel = styled('div')`
+  position: fixed;
+  bottom: 0;
+  background: ${p => p.theme.whiteDark};
+  z-index: ${p => p.theme.zIndex.sidebar};
+  color: #2f2936;
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx]:
```suggestion
  color: ${p => p.theme.sidebar.background};
```

File: src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx [hunk 5]
```diff
@@ -0,0 +1,104 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+import {css} from '@emotion/core';
+import theme from 'app/utils/theme';
```
Comment (matejminar) [inline src/sentry/static/sentry/app/components/sidebar/sidebarPanel.tsx]:
```suggestion

import theme from 'app/utils/theme';
import space from 'app/styles/space';
```

**Review by matejminar — APPROVED** [review]

**Comment by EvanPurkhiser** [comment]
Will update snapshots in a min

**Pull request merged**